GVFS/GVFS.Common/Cleanup/PrefetchStep.cs

```diff
@@ -147,27 +191,34 @@ public static class CommitPrefetcher
                     metadata = new EventMetadata();
                     metadata.Add("path", packPath);
                     metadata.Add(TracingConstants.MessageKey.InfoMessage, $"{nameof(TryGetMaxGoodPrefetchTimestamp)} deleting bad pack file");
-                    tracer.RelatedEvent(EventLevel.Informational, $"{nameof(TryGetMaxGoodPrefetchTimestamp)}_DeleteBadPack", metadata);
-                    if (!fileSystem.TryWaitForDelete(tracer, packPath, IoFailureRetryDelayMS, MaxDeleteRetries, RetryLoggingThreshold))
+                    this.Context.Tracer.RelatedEvent(EventLevel.Informational, $"{nameof(TryGetMaxGoodPrefetchTimestamp)}_DeleteBadPack", metadata);
+
+                    if (!this.Context.FileSystem.TryWaitForDelete(this.Context.Tracer, packPath, IoFailureRetryDelayMS, MaxDeleteRetries, RetryLoggingThreshold))
                     {
                         error = $"Unable to delete {packPath}";
                         return false;
                     }
                 }
             }
 
+            if (this.Stopping)
+            {
+                error = nameof(this.Stopping);
+                return false;

```

Comment:
Is stopping really an error condition?

Reply:
No, but returning `false` results in logging _something_, so it may as well be a message that we are stopping the step early.

I could rename `error` to `message`.

Otherwise, we would need to return an `enum` of return codes, but that seems overkill for something that won't happen often.